docs/events/avasummit/overview.md

```diff
@@ -118,14 +118,14 @@ To use this contract:
 
 ### Configuration
```

Comment:
Do we still need this page for the Avasummit? Can we remove it?

Reply:
Removed